prow/horologium/deployment.yaml

```diff
@@ -20,7 +20,7 @@ spec:
       terminationGracePeriodSeconds: 30
       containers:
       - name: horologium
-        image: gcr.io/k8s-prow/horologium:v20190307-bbad344
+        image: gcr.io/k8s-prow/horologium:v20190402-d65ae756e
```

Comment:
Same as above

Reply:
@JoelSpeed I decided to update the images of components other than deck and hook just under the logic of "might as well"

Reply:
Sounds good